src/helpers/alias.ts

```diff
@@ -1,10 +1,104 @@
+import { uniq } from 'lodash';
 import db from './mysql';
 
-export async function isValidAlias(address: string, alias: string): Promise<boolean> {
-  const thirtyDaysAgo = Math.floor(new Date().getTime() / 1000) - 30 * 24 * 60 * 60;
+const DEFAULT_ALIAS_EXPIRY_DAYS = 30;
 
-  const query =
-    'SELECT address, alias FROM aliases WHERE address = ? AND alias = ? AND created > ? LIMIT 1';
-  const results = await db.queryAsync(query, [address, alias, thirtyDaysAgo]);
-  return !!results[0];
+// These types can always be executed with an alias
+const TYPES_EXECUTABLE_BY_ALIAS = [
+  'follow',
+  'unfollow',
+  'subscribe',
+  'unsubscribe',
+  'profile',
+  'statement'
+] as const;
+
+// These types can be executed with an alias only when enabled in the space settings
+const OPTIONAL_TYPES_EXECUTABLE_BY_ALIAS = [
+  'vote',
+  'vote-array',
+  'vote-string',
+  'proposal',
+  'update-proposal',
+  'delete-proposal'
+] as const;
+
+// These types can be executed with a Starknet alias
+const TYPES_EXECUTABLE_BY_STARKNET_ALIAS = [...OPTIONAL_TYPES_EXECUTABLE_BY_ALIAS] as const;
```

Comment:
Do we need a separate list?  also feel like allowedTypesCache not relavent anymore

Reply:
Maybe we will add more actions just for starknet later ?

`settings` could be added later, since we can now add a starknet address as space admin